src/calculateRentalCost.js

```diff
@@ -1,10 +1,22 @@
+'use strict';
 /**
  * @param {number} days
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCost = days * 40;
```

Comment:
Consider using a constant for the daily rental cost instead of a literal to make the code more maintainable and self-documenting. For example, `const DAILY_RENTAL_COST = 40;`.